src/components/MAPI/apps/ClusterDetailAppListWidgetVersionInspector.tsx

```diff
@@ -202,10 +206,19 @@ const ClusterDetailAppListWidgetVersionInspector: React.FC<IClusterDetailAppList
         setAppUpdateIsLoading(false);
         handleCancelSwitchingVersion();
 
+        const truncatedVersion = truncate(
```

Comment:
Why not use the `<Truncated />` component? You can specify a custom wrapper element for it `<Truncated as='code' />`

There's an example lower down in the file

Reply:
`FlashMessage` is expecting a string for the text input, so I thought it was more straight-forward to use a string. I'll change it to use the `Truncate` component 👍

Reply:
Actually, you're right... we don't allow React components in there, only `<code />` elements. My bad, sorry 👍